### ISSUE TITLE
qcovert will not decrypt simple password-protected files.

### ISSUE THREAD

**Issue by apatheticposture** [covert-encryption/covert#98]
This error occurs when I use **qcovert** and choose **Open Covert File**, an armored or unarmored passphrase-encrypted text file.

The file **can** be decrypted at the command line.

```
Traceback (most recent call last):
File "/home/ap/.local/lib/python3.10/site-packages/covert/gui/app.py", line 104, in decrypt_file
self.decrypt(data)
File "/home/ap/.local/lib/python3.10/site-packages/covert/gui/app.py", line 109, in decrypt
d = DecryptView(self, infile)
File "/home/ap/.local/lib/python3.10/site-packages/covert/gui/decrypt.py", line 35, in __init__
self.decrypt_attempt()
File "/home/ap/.local/lib/python3.10/site-packages/covert/gui/decrypt.py", line 105, in decrypt_attempt
if self.blockstream.header.key:
AttributeError: 'BlockStream' object has no attribute 'header'
```
From here, nothing happens - we are at the main screen.

Example (armored) file --

Password is oakpigartbecome

`Unz4TxF16QTcqbtEbpc56IgOm+gmaIbD+/dpcIVeTc1PH8b9DQevQ5SsMum99yhgRAByRKJ360tDCo8yLEogl3kGNCw3ISbH161ux7EVvQZCPJF6JfyVQ4Cq/6fCd1fBK2eOqKXf4w6atfIS+NCP8V2eBbfJrog0w73jVQ`

**Comment by covert-encryption** [comment]
The GUI is not quite up to date in current release, as we lost our GUI developer. You may install 0.6 if you wish to try it but I will try to get it back in operation fairly soon. Notice that it also lacks some features, most importantly those of key storage (idstore) and forward secrecy. Thus for now I recommend the CLI version.

Will update this bug once the GUI is once again operational.

**Comment by covert-encryption** [comment]
This is fixed in #99.
